directories/src/redux/services/brokernode.js

```diff
@@ -0,0 +1,107 @@
+import axios from "axios";
+import { API_HEADERS, API_ROOT_URL, API_REQUEST_ERROR } from "../../config/";
+
+const axiosInstance = axios.create({
+  timeout: 200000
+});
+
+const registerWebnode = address =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/supply/webnodes`,
+      data: { address }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST register" + error);
+      reject(API_REQUEST_ERROR);
+    });
+  });
+
+
+const requestAddressPoW = (currentList, to) =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "POST",
+      url: `${API_ROOT_URL}/api/v1/demand/transactions/${to}`,
+      data: {currentList}
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("POST request" + error);
+      reject(API_REQUEST_ERROR);
+    });
+  });
+
+const completeAddressPoW = (txid, trytes, to) =>
+  new Promise((resolve, reject) => {
+    axios({
+      method: "PUT",
+      url: `${API_ROOT_URL}/api/v1/demand/transactions/${txid}/${to}`,
+      data: { trytes }
+    })
+    .then(data => {
+      resolve({ data });
+    })
+    .catch(error => {
+      console.log("PUT complete " + error);
+      reject(API_REQUEST_ERROR);
+    });
+  });
+
+const broadcastToHooks = (trytes, nodes) => {
```

Comment:
We will no longer need to broadcast to the hooknodes anymore.  For the genesis hash and brokernode address transactions, the brokernode will do the broadcast for us when we make the API call to complete the transaction.   

For the treasure hunting, I think we said we'd just use the brokernode's Iota API to do the broadcasting.  So we'll still need a method for that but it won't be calling http://${node}:3000/broadcast/